includes/dbFacile.php

```diff
@@ -24,10 +24,10 @@
 function dbQuery($sql, $parameters = array()) {
 	global $fullSql, $debug;
 	$fullSql = dbMakeQuery($sql, $parameters);
-        if($debug) { 
-          print Console_Color::convert("\nSQL[%y".$fullSql."%n] ");
-          #echo("\nSQL[".$fullSql."] "); 
-        }
+        //if($debug) { 
```

Comment:
This needs to stay in really, it's for running poller and discovery in debug mode.